.github/workflows/test_release_packages.yml

```diff
@@ -0,0 +1,115 @@
+name: Test release packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+      tag:
+        type: string
+
+jobs:
+  generate_release_matrix:
```

Comment:
I am not sure the job need to generate the release matrix to test for itself. This could be an input form the prior job. With this it's also more easy to trigger exactly one run on a specific system and for a specific GPU.